core/iwasm/aot/aot_runtime.h

```diff
@@ -130,6 +130,9 @@ typedef struct LocalRefFlag {
 typedef struct AOTModule {
     uint32 module_type;
 
+    /* the package version, should be AOT_CURRENT_VERSION */
```

Comment:
Same as wasm.h, how about somewhat like the file version read from the AOT file?

Reply:
This seems to contradict your earlier comment:
https://github.com/bytecodealliance/wasm-micro-runtime/pull/3601#discussion_r1669519462

Reply:
I will change it anyway.

Reply:
> This seems to contradict your earlier comment: [#3601 (comment)](https://github.com/bytecodealliance/wasm-micro-runtime/pull/3601#discussion_r1669519462)

Yes, but I am not very sure whether AOT runtime of newer version will support AOT file of some older version in the future. Thanks for the update.